content/en/blog/_posts/2022-08-23-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,173 @@
+
+### Promoted CRD Validation Expression Language to Beta
+
+[CRD Validation Expression Language](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2876-crd-validation-expression-language/README.mdv) is promoted to beta, which makes it possible to declare how custom resources are validated using the [Common Expression Language (CEL)](https://github.com/google/cel-spec). Please see the [validation rules](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#validation-rules) guide.
+
+### Promoted Server Side Unknown Field Validation to Beta
+
+Promoted the `ServerSideFieldValidation` feature gate to beta (on by default). This allows optionally triggering schema validation on the API server that errors when unknown fields are detected. This allows the removal of client-side validation from kubectl while maintaining the same core functionality of erroring out on requests that contain unknown or invalid fields.
+
+###  Introduced KMS v2
+
+Introduce KMS v2alpha1 API to add performance, rotation, and observability improvements. Encrypt data at rest (ie Kubernetes `Secrets`) with DEK using AES-GCM instead of AES-CBC for kms data encryption. No user action is required. Reads with AES-GCM and AES-CBC will continue to be allowed. See the guide [Using a KMS provider for data encryption](https://kubernetes.io/docs/tasks/administer-cluster/kms-provider/) for more information.
+
+### Removed AKS and GKE authentication from kubectl
+
+The `gcp` and `azure` auth plugins have been removed from client-go and kubectl. See [AKS](https://github.com/Azure/kubelogin) and [GKE](https://cloud.google.com/blog/products/containers-kubernetes/kubectl-auth-changes-in-gke) documentation for details about the cloud-specific replacements.
+
+## Other Updates
+
+### Graduations to Stable
+
+This release includes a total of thirteen enhancements promoted to stable: 
+
+* [Ephemeral Containers](https://github.com/kubernetes/enhancements/issues/277)
+* [Local Ephemeral Storage Resource Management](https://github.com/kubernetes/enhancements/issues/361)
+* [CSI Ephemeral Volumes](https://github.com/kubernetes/enhancements/issues/596)
+* [CSI Migration - Core](https://github.com/kubernetes/enhancements/issues/625) 
+* [Graduate the kube-scheduler ComponentConfig to GA](https://github.com/kubernetes/enhancements/issues/785)
+* [CSI Migration - AWS](https://github.com/kubernetes/enhancements/issues/1487)
+* [CSI Migration - GCE](https://github.com/kubernetes/enhancements/issues/1488) 
```

Comment:
Could we group the CSI Migration graduations together?

Reply:
I thought about that because it definitely does look a little overwhelming, but then we lose the ability to cleanly link to the KEP for each change. Totally open to a better organizational suggestion here. Maybe they just go under their own heading so it's pulled out a bit more?